pennylane_lightning/src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(pennylane_lightning_tests)
+
+set(CMAKE_CXX_STANDARD 11)
+find_package(OpenMP)
+
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
+)
+# For Windows: Prevent overriding the parent project's compiler/linker settings
+set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+FetchContent_MakeAvailable(googletest)
+enable_testing()
```

Comment:
Is this necessary given that we've already enabled testing in the parent CMakeLists.txt?

Reply:
Hi Mandrenkov,

thank you for all the comments. I added this to support the build and test of `tests` directory independently, but it can be removed if it is not a common use case.

Reply:
Keeping this in both is a good call, as they can be run independently. I'm in favour of keeping it.

Reply:
Sure.  I guess it doesn't hurt to have it in both places.